core/modules/startup/windows.js

```diff
@@ -33,8 +33,16 @@ exports.startup = function() {
 			height = paramObject.height || "600",
 			variables = $tw.utils.extend({},paramObject,{currentTiddler: title});
 		// Open the window
-		var srcWindow = window.open("","external-" + title,"scrollbars,width=" + width + ",height=" + height),
```

Comment:
It looks like this change is a separate fix for a problem with opening a new window. Perhaps it's better in it's own PR so that it doesn't get lost?

Reply:
@Jermolene thanks, I've removed this - I'll open a separate PR